digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/dao/ProjectDAO.scala

```diff
@@ -488,6 +488,13 @@ object ProjectDAO {
     Q.updateNA(sql).execute
   }
 
+  def updateProjectLinkNumberingAndTrack(projectId: Long, projectLinkIds: Seq[Long], newRoadNumber: Long, newRoadPart: Long, newTrackCode:Long, userName: String ): Unit ={
```

Comment:
i am wondering if we should copy new numbering and track to projectlink seq links and use updateProjectLinksToDB to reduce amount of sql code and prefer prepared statements?

Reply:
actually it seems that we are not using this method anywhere